setup.py

```diff
@@ -17,9 +17,8 @@
 
 
 # Check this Python version is supported
-if any([v < (2, 6), (3,) < v < (3, 7)]):
-    raise Exception("Unsupported Python version %d.%d. Requires Python >= 2.7 "
-                    "or >= 3.7." % v[:2])
+if v > (3, 7):
```

Comment:
```suggestion
if v < (3, 7):
```

Reply:
Actually we need 3.11 compatibility `if v < (3, 11):`